src/extlibs/enlsvg/IndirectHeap.cpp

```diff
@@ -0,0 +1,107 @@
+#include "extlibs/enlsvg/IndirectHeap.h"
```

Comment:
we have indirect_heap.cpp which is a successor to this file, so you can delete this file